bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RegionAwareEnsemblePlacementPolicy.java

```diff
@@ -388,9 +388,20 @@ public PlacementResult<List<BookieId>> newEnsemble(int ensembleSize, int writeQu
                 remainingEnsembleBeforeIteration = remainingEnsemble;
                 int regionsToAllocate = numRemainingRegions;
                 int startRegionIndex = lastRegionIndex % numRegionsAvailable;
+                int localRegionIndex = -1;
```

Comment:
Maybe a more clear way to do this is to remove the localRegion from the availableRegions then everything happen smoothly with original logic.

Reply:
Maybe not. If so, the `localRegion` node can't pick again. If we want to pick 5 nodes, and there are only two regions. If we remove `localRegion`, then we can only pick nodes from the remaining one region.